src/app/component/core/search-popup/search-popup.component.html

```diff
@@ -11,47 +11,49 @@
     </div>
   </div>
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/82374788-382c0600-9a28-11ea-9244-dd518f769bff.png)

Reply:
Search results area should has width 100%, and results should be aligned to the center

Reply:
![image](https://user-images.githubusercontent.com/54113846/82375070-b092c700-9a28-11ea-86ba-848915445783.png)

Also, please, open eco-news tab -> scroll down  -> open search -> enter something

Reply:
You don't see anything, 'cause block with class '.search-content-wrapper' is still on the top, as a result you should use position fixed for '.search-content-wrapper'

Reply:
fixed